src/Resources/Resources/help/New-AzDeploymentWhatIf.md

```diff
@@ -0,0 +1,466 @@
+ [-Mode <DeploymentMode>] [-ResultFormat <WhatIfResultFormat>] [-AsJob] -TemplateObject <Hashtable>
+ [-SkipTemplateParameterPrompt] [-ApiVersion <String>] [-Pre] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceGroupWithTemplateObjectAndParameterObject
+```
+New-AzDeploymentWhatIf [-Name <String>] -ScopeType <DeploymentWhatIfScopeType> -ResourceGroupName <String>
+ [-Mode <DeploymentMode>] [-ResultFormat <WhatIfResultFormat>] [-AsJob] -TemplateParameterObject <Hashtable>
+ -TemplateObject <Hashtable> [-SkipTemplateParameterPrompt] [-ApiVersion <String>] [-Pre]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceGroupWithTemplateObjectAndParameterFile
+```
+New-AzDeploymentWhatIf [-Name <String>] -ScopeType <DeploymentWhatIfScopeType> -ResourceGroupName <String>
+ [-Mode <DeploymentMode>] [-ResultFormat <WhatIfResultFormat>] [-AsJob] -TemplateParameterFile <String>
+ -TemplateObject <Hashtable> [-SkipTemplateParameterPrompt] [-ApiVersion <String>] [-Pre]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceGroupWithTemplateFileAndParameterObject
+```
+New-AzDeploymentWhatIf [-Name <String>] -ScopeType <DeploymentWhatIfScopeType> -ResourceGroupName <String>
+ [-Mode <DeploymentMode>] [-ResultFormat <WhatIfResultFormat>] [-AsJob] -TemplateParameterObject <Hashtable>
+ -TemplateFile <String> [-SkipTemplateParameterPrompt] [-ApiVersion <String>] [-Pre]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceGroupWithTemplateFileAndParameterFile
+```
+New-AzDeploymentWhatIf [-Name <String>] -ScopeType <DeploymentWhatIfScopeType> -ResourceGroupName <String>
+ [-Mode <DeploymentMode>] [-ResultFormat <WhatIfResultFormat>] [-AsJob] -TemplateParameterFile <String>
+ -TemplateFile <String> [-SkipTemplateParameterPrompt] [-ApiVersion <String>] [-Pre]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceGroupWithTemplateObjectAndNoParameters
+```
+New-AzDeploymentWhatIf [-Name <String>] -ScopeType <DeploymentWhatIfScopeType> -ResourceGroupName <String>
+ [-Mode <DeploymentMode>] [-ResultFormat <WhatIfResultFormat>] [-AsJob] -TemplateObject <Hashtable>
+ [-SkipTemplateParameterPrompt] [-ApiVersion <String>] [-Pre] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceGroupWithTemplateFileWithAndNoParameters
+```
+New-AzDeploymentWhatIf [-Name <String>] -ScopeType <DeploymentWhatIfScopeType> -ResourceGroupName <String>
+ [-Mode <DeploymentMode>] [-ResultFormat <WhatIfResultFormat>] [-AsJob] -TemplateFile <String>
+ [-SkipTemplateParameterPrompt] [-ApiVersion <String>] [-Pre] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
```

Comment:
>DESCRIPTION [](start = 3, length = 11)

go through the descriptions with Tom/Jonathan :)

Reply:
Sure thing!

Reply:
Sent an email to them to ask help :)